auditbeat/auditbeat.reference.yml

```diff
@@ -148,6 +148,50 @@ auditbeat.modules:
     # if the number of events stored in the queue is < min_flush_events.
     #flush.timeout: 1s
 
+  # The spool queue will store events in a local spool file, before
```

Comment:
I assume only 1 queue can be used at the same time. Should we mention that in a comment on line 130?

Reply:
We need to?

Reply:
I mainly brought it up as people keep trying to use 2 outputs in Beats and they will for queues. So having it documented makes it easy to point people to it. So they know it already from the docs before running the docs.